zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -69,6 +69,28 @@ impl<F: Field> SignVerifyChip<F> {
             _marker: PhantomData,
         }
     }
+
+    /// Return the minimum number of rows required to prove an input of a
+    /// particular size.
+    pub fn min_num_rows(num_verif: usize) -> usize {
+        // The values rows_ecc_chip_aux, rows_ecdsa_chip_verification and
+        // rows_ecdsa_chip_verification have been obtained from log debugs while running
+        // the tx circuit with max_txs=1. For example:
+        // `RUST_LOG=debug RUST_BACKTRACE=1 cargo test tx_circuit_1tx_1max_tx --release
+        // --all-features -- --nocapture`
+        // The value rows_range_chip_table has been optained by patching the halo2
+        // library to report the number of rows used in the range chip table
+        // region. TODO: Figure out a way to get these numbers automatically.
+        let rows_range_chip_table = 295188;
+        let rows_ecc_chip_aux = 226;
+        let rows_ecdsa_chip_verification = 140360;
+        let rows_signature_address_verify = 76;
```

Comment:
We should really prioritize the work on getting this automatically. Otherwise, any PR can invalidate this!

Reply:
Can we fill an issue for this @ed255??